src/css/index.css

```diff
@@ -60,6 +60,12 @@ body {
     cursor: not-allowed;
 }
 
+.button.varButtonDisabled{
```

Comment:
Because our buttons are divs with a "button" class there is no way to disable variable reference buttons other than to set their pointer events to none. This is something we don't want to to do all the other toolbox buttons hence a separate class is required.

This will no longer be needed once we refactor how variable reference buttons work.

Reply:
Thanks for explaining these. yes, we will do as you said.